src/modules/commander/commander.cpp

```diff
@@ -1198,6 +1198,7 @@ int commander_thread_main(int argc, char *argv[])
 	bool was_armed = false;
 
 	bool startup_in_hil = false;
+	bool auto_disarm_enable = false; //the flag indicate enable auto disarm when the vehicle state from inair to land
```

Comment:
@dagar If that, the code will be not different from before fixing the bug, so the vehicle will not be armed still.